pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -633,17 +578,17 @@ def get_detector_par(self, det, hdu=None):
             specaxis        = 0,
             specflip        = False,
             spatflip        = False,
-            platescale      = 0.16, # average from order 17 and order 30, see manual
+            platescale      = 0.154, # average from order 17 and order 30, see manual
             darkcurr        = 0.0,  # e-/pixel/hour
             saturation      = 65535.,
             nonlinear       = 0.86,
             mincounts       = -1e10,
             numamplifiers   = 1,
-            gain            = np.atleast_1d(0.595), # FITS format is flipped: PrimaryHDU  (2106, 4000) w/respect to Python
-            ronoise         = np.atleast_1d(3.1), # raw unbinned images are (4000,2106) (spec, spat)
-            datasec=np.atleast_1d('[:,11:2058]'),  # pre and oscan are in the spatial direction
-            oscansec=np.atleast_1d('[:,2059:2106]'),
-        )
+            gain            = np.atleast_1d(0.64),   # Assumes high gain. TODO: grab this from the header
```

Comment:
There are a few instruments that pull this kind of information from the header.  The one that comes to mind is KCWI.

Reply:
I noticed that the binning was already being grabbed via a single line just above, so I followed that example. Let me know if you think it should be done differently.